terraform/datadog/cloud_controller.tf

```diff
@@ -0,0 +1,129 @@
+resource "datadog_monitor" "cc_api_master_process_running" {
+  name                = "${format("%s Cloud Controller API master process running", var.env)}"
+  type                = "service check"
+  message             = "Cloud Controller API master process is not running. Check deployment state."
+  escalation_message  = "Cloud Controller API master process is still not running. Check deployment state."
+  notify_no_data      = false
+  require_full_window = true
+
+  query = "${format("'process.up'.over('bosh-deployment:%s','process:cc_api_master').last(4).count_by_status()", var.env)}"
```

Comment:
I think there is a bug in DD. It shows more hosts in the  "Monitored Hosts" but the right hosts in "Monitor History"